.env-devel

```diff
@@ -154,7 +154,7 @@ SMTP_PASSWORD=it_doesnt_matter
 SMTP_PROTOCOL=UNENCRYPTED
 
 SIMCORE_SERVICES_NETWORK_NAME=interactive_services_subnet
-SIMCORE_INTERACTIVE_NETWORK_SUBNET=172.26.0.0/16
+SIMCORE_INTERACTIVE_NETWORK_SUBNET=172.8.0.0/16
```

Comment:
Why are you changing this in this PR? please ask @matusdrobuliak66 since they have been tuning this value in various PRs. perhaps you did a bad merge?

Reply:
@elisabettai this was now fixed by @matusdrobuliak66 in master. so I think you can revert that file and update again from the master branch

Reply:
I should have not staged this change, thanks for noticing. It is fixed now.